NEWS.md

```diff
@@ -1,3 +1,7 @@
+**Version 2.28.0 2024-09-62**
+- Metadata for layer items like variables, projections, dimensions and styles are now stored in a metadatatable. This can be disabled via the `enablemetadatacache` property in [Settings](doc/configuration/Settings.md).
```

Comment:
```suggestion
- Metadata for layer items like variables, projections, dimensions and styles are now stored in a database table. This can be disabled via the `enablemetadatacache` property in [Settings](doc/configuration/Settings.md).
```

Reply:
Good suggestion, changed